README/README.md

```diff
@@ -0,0 +1,50 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-World Website
+
+> This is my first code Exercise in Microverse
+
+![screenshot](./app_screenshot.png)
+
+
+## Built With
+
+- HTML & CSS
+
+## Live Demo
+
+[Live Demo Link](https://felipeg005.github.io/Hello-World/)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps
+
+### Prerequisites
+
+- Browser (Chrome,Firefoxe,IE,Opera,Safari)
+
+### Usage
+
+- Open Html.index file with your browser
+
+
+## Authors
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/Felipeg005/)
+- Twitter: [@twitterhandle](https://twitter.com/JuanFGT05)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/juan-felipe-gomez-tobon/)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thanks to Microverse for make it possible!
+
+## 📝 License
+
```

Comment:
Good job writing a descriptive readme for your project :+1: 
Please move it up a directory to the main project path/directory ( the source directory ) so that Github can display the file as the documentation for your project.
Readme should not be in a folder but outside in your project directory/folder.

Reply:
Perfect, that's right. Thank you SamroodAli for your review, now, I updated the changes and ask for another review.